src/components_redux/Radios.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+
+export default class Radios extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      type: " "
+    };
+  }
+  optionsShow = () => {
+    this.setState({ showComponet: !this.state.showComponet });
+  };
+  radioChange = event => {
+    this.setState({
+      type: event.target.name
+    });
+  };
+  render() {
+    return (
+      <div>
+        <label>Type</label>
+        <label>
+          <input
+            type="radio"
+            name="text"
```

Comment:
instead of name you should use value - see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

name is the "key" and value is the value of the element

Reply:
Okay, I will change this.